website/src/components/TreeView/data.js

```diff
@@ -0,0 +1,357 @@
+export default {
+  name: 'sample_registration',
+  required: true,
+  fields: [
+    { name: 'program_id', required: true },
+    { name: 'submitter_donor_id', required: true },
+    { name: 'gender', required: true },
+    { name: 'submitter_specimen_id', required: true },
+    { name: 'specimen_tissue_source', required: true },
+    { name: 'tumour_normal_designation', required: true },
+    { name: 'submitter_sample_id', required: true },
+    { name: 'sample_type', required: true },
+  ],
+  children: [
+    {
+      name: 'specimen',
```

Comment:
is this file still used?

Reply:
Not actively. I'd like to leave it for now just to have a sample of what the tree is expecting.
I'll rename to dummy_data or something to make it less confusing